resources/todomvc/vanilla-examples/javascript-es5/src/view.js

```diff
@@ -31,12 +31,11 @@
     View.prototype._removeItem = function (id) {
         var elem = qs(`[data-id="${id}"]`);
 
-        if (elem)
-            this.$todoList.removeChild(elem);
+        if (elem) this.$todoList.removeChild(elem);
     };
 
     View.prototype._clearCompletedButton = function (completedCount, visible) {
-        this.$clearCompleted.innerHTML = this.template.clearCompletedButton(completedCount);
+        this.$clearCompleted.textContent = this.template.clearCompletedButton(completedCount);
```

Comment:
`clearCompletedButton` is always a simple string, so I replaced this one with just `textContent`.